src/CodeAnalysis.ruleset

```diff
@@ -1,42 +1,87 @@
+    <Rule Id="CA1303" Action="None" /> <!-- Do not pass literals as localized parameters -->
+    <Rule Id="CA1304" Action="None" /> <!-- Specify CultureInfo -->
+    <Rule Id="CA1305" Action="None" /> <!-- Specify IFormatProvider -->
+    <Rule Id="CA1307" Action="None" /> <!-- Specify StringComparison -->
+    <Rule Id="CA1308" Action="None" /> <!-- Normalize strings to upper case -->
+    <Rule Id="CA1401" Action="None" /> <!-- P/Invoke method should not be visible -->
+    <Rule Id="CA1507" Action="None" /> <!-- Use nameof to express symbol names -->
+    <Rule Id="CA1707" Action="None" /> <!-- Identifers should not contain underscores -->
+    <Rule Id="CA1710" Action="None" /> <!-- Identifers should have correct suffix -->
+    <Rule Id="CA1712" Action="None" /> <!-- Do not prefix enum values with type name -->
+    <Rule Id="CA1714" Action="None" /> <!-- Flags enums should have plural names -->
     <Rule Id="CA1715" Action="None" /> <!-- Type parameters names should be prefixed with T -->
+    <Rule Id="CA1716" Action="None" /> <!-- Identifiers should not match keywords -->
+    <Rule Id="CA1717" Action="None" /> <!-- Only FlagsAttribute enums should have plural names -->
+    <Rule Id="CA1720" Action="None" /> <!-- Identifier contains type name -->
+    <Rule Id="CA1721" Action="None" /> <!-- Property names should not match get methods -->
+    <Rule Id="CA1724" Action="None" /> <!-- Type names should not match namespaces -->
+    <Rule Id="CA1801" Action="None" /> <!-- Review unused parameters -->
+    <Rule Id="CA1802" Action="None" /> <!-- Use literals where appropriate -->
+    <Rule Id="CA1806" Action="None" /> <!-- Do not ignore method results -->
+    <Rule Id="CA1810" Action="None" /> <!-- Initialize reference type static fields inline -->
+    <Rule Id="CA1812" Action="None" /> <!-- Avoid uninstantiated internal classes -->
+    <Rule Id="CA1814" Action="None" /> <!-- Prefer jagged arrays over multidimensional -->
+    <Rule Id="CA1815" Action="None" /> <!-- Override equals and operator equals on value types -->
+    <Rule Id="CA1816" Action="None" /> <!-- Dispose methods should call SuppressFinalize -->
+    <Rule Id="CA1819" Action="None" /> <!-- Properties should not return arrays -->
+    <Rule Id="CA1820" Action="None" /> <!-- Test for empty strings using string length -->
+    <Rule Id="CA1821" Action="None" /> <!-- Remove empty finalizers --> <!-- Disable analyzer warning 1821 until https://github.com/dotnet/roslyn-analyzers/issues/1804 is resolved -->
+    <Rule Id="CA1822" Action="None" /> <!-- Mark members as static -->
+    <Rule Id="CA1823" Action="None" /> <!-- Avoid unused private fields -->
+    <Rule Id="CA1825" Action="None" /> <!-- Avoid zero-length array allocations. -->
+    <Rule Id="CA2000" Action="None" /> <!-- Dispose objects before losing scope -->
+    <Rule Id="CA2002" Action="Error" /> <!-- Do not lock on objects with weak identity -->
```

Comment:
Should this instead just be omitted from the list? It would error if we don't specify anything for that rule, right?

Reply:
Probably. Will double check and then remove.